xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1186,7 +1186,7 @@ bool CGUIWindowVideoBase::OnPlayAndQueueMedia(const CFileItemPtr& item, const st
 
 void CGUIWindowVideoBase::PlayMovie(const CFileItem *item, const std::string &player)
 {
-  if(m_thumbLoader.IsLoading())
+  if (m_thumbLoader.IsLoading())
```

Comment:
Unrelated change (but still welcome). Don't worry about this now, but as the PR progresses you'll incorporate feed and combine the commits into a clean history, and at that point you'll want to split this out into a second commit.

Keep an eye on those "lines added" and "lines deleted" metrics. The main commit should ideally have 0 lines deleted. This indicates to me that no existing code is touched, making the feature much safer and more desirable.